psi4/src/psi4/libmints/extern.cc

```diff
@@ -315,6 +315,10 @@ double ExternalPotential::computeNuclearEnergy(std::shared_ptr<Molecule> mol) {
         double yA = mol->y(A);
         double zA = mol->z(A);
         double ZA = mol->Z(A);
+        // skip Ghost interaction
+        if (ZA == 0) {
```

Comment:
Because `ZA` is a double it is better to avoid `==` and go with a `if (ZA > 0) {` loop around the computations instead.